### PR TITLE
feat: clear stamp cache on data update

### PR DESCRIPTION
**Pull request by wa0x6e**
Related to https://discord.com/channels/955773041898573854/1031838169282392144/1249097030094291066

This PR will call stamp service and clear the relevant cache on avatar update

Cache will be cleared when updating:

- user's avatar
- spaces' avatar

### CODE REVIEW

File: src/writer/profile.ts [hunk 1]
```diff
@@ -25,4 +29,6 @@ export async function action(message, ipfs): Promise<void> {
   };
 
   await db.queryAsync('REPLACE INTO users SET ?', params);
+
+  if (profile.avatar !== existingProfile.avatar) clearStampCache('avatar', message.from);
```
Comment (ChaituVR) [inline src/writer/profile.ts:33]: Maybe better to wait until we finish the fetch, else user may see old avatar for some time,

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/writer/profile.ts (hunk 1 above)
Comment (wa0x6e) [reply src/writer/profile.ts:33]: How could we achieve that ? What do you mean by finish the fetch ?

This is the method used by the UI (directly trigger this request after profile update), and I did notice that sometimes, the cache is not clearing correctly.

Comment (ChaituVR) [reply src/writer/profile.ts:33]: I mean to wait until the cache is cleared.

**Review by ChaituVR — APPROVED** [review]
tAck

**Pull request merged**